pkg/runner/runner.go

```diff
@@ -44,23 +44,22 @@ alive:
 	for {
 		select {
 		case <-sigTerm:
-			log.Println("octo-proxy interrupted")
+			log.Warn().Msg("octo-proxy interrupted")
 
 			octo.Lock()
 			shutdown(octo.Proxies, srv)
 			octo.Unlock()
 
 			break alive
 		case <-sigReload:
-			log.Println("octo-proxy reload trigerred")
+			log.Warn().Msg("octo-proxy reload triggered")
```

Comment:
can you set this as `Info`? because nothing went wrong or might wrong in the near future

Reply:
Done